### DISCUSSION TITLE
MQTT im ioBroker

### DISCUSSION THREAD

**Discussion by RasputinBerry** [jomjol/AI-on-the-edge-device#1394] [Q&A]
Danke für dieses Projekt !! ... eigentlich klappt alles aber ....:
- Dies sind meine MQTT Einstellungen ...
![MQTTEinstellung](https://user-images.githubusercontent.com/56359831/203305538-11a4e2cf-c4cd-4462-99cd-9470ef1cc573.jpg)
- und das kommt in den ioBroker Objekten an ...
![MQTT](https://user-images.githubusercontent.com/56359831/203305641-599677c4-0cd2-4add-9e4a-faf1dc8e853a.jpg)

Die gelbe Markierung im main-Ordner ist der aktuelle Wert. Dieser Ordner sollte doch direkt unter "WasserZähler" stehen, oder ? 
Die orange Markierung ist "Müll" ... wo kommt der her ? Was habe ich falsch gemacht ? (Egal ob das Flag auf true / false steht)
Vielen Dank für die Hilfe (zu der wahrscheinlichen Anfänger-Frage ...) ...

**Comment by caco3** [comment]
Welche Version verwendest Du?

Bei mir sieht es so aus:

![image](https://user-images.githubusercontent.com/1783586/203406570-3c0fd49f-495e-45b7-8a27-650d845b53d5.png)

Das ist basierend auf rolling, da haben wir einige topics hinzugefügt, aber die Struktur war schon vorher so

**Comment by RasputinBerry** [comment]
... ich nehme mal an, mit "Version" meinst Du diesen Screenshot:
![Version](https://user-images.githubusercontent.com/56359831/203509007-c63ea9d6-d6dc-4ffb-99e5-72027349118a.jpg)

Und wenn dein Screenshot deine MQTT Einträge / Struktur zeigt und ich die mit meinen (automatisch) angelegten Einträgen vergleiche kann ich nicht wirklich eine Ähnlichkeit feststellen. 
Zwischenzeitlich habe ich die "orangen" Anteile gelöscht, seitdem ist nix neues dazu gekommen. Ich habe seitdem auch den MQTT Adpater oder den Zähler nicht neu gestartet. Ich nehme an, falls ich das tun würde sind die "überflüssigen" Einträge wieder da ...

**Reply by caco3** [reply]
Ich bin nach wie vor der Meinung, dass diese orangen EInträge nicht vom device kommen.
Denn sonst müsste ich die im MQTT ja auch sehen!
Ev. etwas von ioBroker?

Versuchs sonst mal mit dem latest rolling: https://github.com/jomjol/AI-on-the-edge-device/actions?query=branch%3Arolling

**Comment by RasputinBerry** [comment]
OK ... ich entnehme der bisherigen Diskussion, das nicht **ICH** das Problem bin, sondern das da irgendwo irgendwas passiert das nicht eindeutig zu identifizieren ist ....
Da meine MQTT und ioBroker Kenntnisse überschaubar sind würde ich an dieser Stelle das Problem gedanklich auf "Stand-by" stellen und auf die nächste stabile Device Version warten. Jetzt auf sowas wie "latest rolling" zu wechseln würde mich schlicht überfordern ... sorry.

So wie es jetzt (mit dem orangen Schönheitsfehler) läuft bin ich (erstmal) zufrieden ... "Never change a running system" .... (in der MQTT Instanz hängen noch andere Geräte , und die funktionieren ...) 

Danke soweit für deine Mühe, ich beobachte weiter ....
